py3-python-daemon.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: py3-python-daemon
+  version: 3.1.0
+  epoch: 0
+  description: Library to implement a well-behaved Unix daemon process.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: python-daemon
+  import: daemon
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '300'
```

Comment:
why is 3.13 is `300`?

Reply:
we set `provider-priority: ${{range.value}}` in the subpackage and we still are defaulting the package to 3.12 version of python. In future, we will be bumping all of that to default to python-3.13. 

So, TLDR is that we want that if users install the package then they get the 3.12 version of python-daemon package. 

```bash
docker run --name wolfi --rm -it cgr.dev/chainguard/wolfi-base:latest sh
/ # apk add py3-semver
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
(1/14) Installing py3-pip-wheel (24.3.1-r0)
(2/14) Installing libbz2-1 (1.0.8-r9)
(3/14) Installing libexpat1 (2.6.4-r0)
(4/14) Installing libffi (3.4.6-r5)
(5/14) Installing gdbm (1.24-r1)
(6/14) Installing xz (5.6.3-r2)
(7/14) Installing libstdc++ (14.2.0-r5)
(8/14) Installing mpdecimal (4.0.0-r3)
(9/14) Installing ncurses-terminfo-base (6.5_p20241006-r4)
(10/14) Installing ncurses (6.5_p20241006-r4)
(11/14) Installing readline (8.2.13-r1)
(12/14) Installing sqlite-libs (3.47.0-r0)
(13/14) Installing python-3.12-base (3.12.7-r1)
(14/14) Installing py3.12-semver (3.0.2-r4)
Executing glibc-2.40-r3.trigger
Executing busybox-1.37.0-r0.trigger
OK: 63 MiB in 29 packages

/ # ls /usr/lib/python3.12/site-packages/semver
__about__.py    __init__.py     __main__.py     __pycache__     _deprecated.py  _types.py       cli.py          py.typed        version.py
```